go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/mimir
 
-go 1.17
+go 1.18
```

Comment:
Is there any need to require Go 1.18 for the Mimir module though? Should think that's independent of upgrading to Go 1.18 for ourselves?

Reply:
Reverted: https://github.com/grafana/mimir/pull/2400/commits/348a8a7230503e798d5d572e47555a82e8ae8ef0

Reply:
There is a new package in 1.18 `net/netip` that I'd like to make use of in dskit.  The thought was that if all the projects were updated to 1.18 we could bump the version in dskit also.

Reply:
See: https://github.com/grafana/mimir/pull/2550